docs/gke/configs/deploy.sh

```diff
@@ -18,15 +18,12 @@ export PROJECT=${PROJECT:-}
 export DEPLOYMENT_NAME=${DEPLOYMENT_NAME:-}
 export ZONE=${ZONE:-}
 export CONFIG_FILE=${CONFIG_FILE:-}
-export CLIENT_ID=${CLIENT_ID:-}
-export CLIENT_SECRET=${CLIENT_SECRET:-}
 
 if [ -z "${PROJECT}" ] || \
    [ -z "${DEPLOYMENT_NAME}" ] || \
    [ -z "${ZONE}" ] || \
-   [ -z "${CONFIG_FILE}" ] || \
-   [ -z "${CLIENT_ID}" ] || \
-   [ -z "${CLIENT_SECRET}" ]; then
+   [ -z "${EMAIL_ADDRESS}" ] || \
```

Comment:
I'm guessing the email address doesn't actually matter?
So lets not make EMAIL_ADDRESS required; lets set it to some default that makes it clear its some dummy value
e.g.
johndoe@acme.com

This way its one less thing and parameter we have to set. And we can get rid of the extra secret below.

We could make emailAddress a parameter of the component and then just set the environment variable directly on the deployment so we don't have to create a config map.